astropy/coordinates/sky_coordinate.py

```diff
@@ -1272,7 +1274,7 @@ def directional_offset_by(self, position_angle, separation):
         See Also
         --------
         position_angle : inverse operation for the ``position_angle`` component
-        separation : inverse operation for the ``separation`` component
+        :meth:`~astropy.coordinates.BaseCoordinateFrame.separation` : inverse operation for the ``separation`` component
```

Comment:
See Also is a special numpydoc directive, not sure if you need the `:meth:` stuff here.